test/unit/src/common/actions/t_repositories.js

```diff
@@ -0,0 +1,131 @@
+import expect from 'expect';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import nock from 'nock';
+import { isFSA } from 'flux-standard-action';
+
+import { conf } from '../../../../../src/server/helpers/config';
+
+import {
+  fetchUserRepositories,
+  setRepositories,
+  fetchRepositoriesError
+} from '../../../../../src/common/actions/repositories';
+import * as ActionTypes from '../../../../../src/common/actions/repositories';
+
+const middlewares = [ thunk ];
+const mockStore = configureMockStore(middlewares);
+
+describe('repositories actions', () => {
+  const initialState = {
+    isFetching: false,
+    success: false,
+    error: null,
+    repos: null
+  };
+
+  let store;
+  let action;
+
+  beforeEach(() => {
+    store = mockStore(initialState);
+  });
+
+  context('setRepositories', () => {
```

Comment:
fwiw, arrow syntax in mocha tests is probably an antipattern (and one I might have introduced!) because you can't avail yourself of skip, retries, or other methods